invenio_jobs/jobs.py

```diff
@@ -39,17 +45,29 @@ def create(
             ),
         )
 
-    @abstractmethod
-    def default_args(self, *args, **kwargs):
-        """Abstract method to enforce implementing default arguments."""
+    @classmethod
+    def get_task_args(cls, job_obj, **kwargs):
```

Comment:
This should be the public method to re-implement as needed:
```suggestion
    def build_task_arguments(cls, job_obj, **kwargs):
```
I think that it has to be a cls method.